tests/cbmc/proofs/prepare.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+import os
+import subprocess
+import sys
+
+
+cmd = ["make", "-B", "cbmc-batch.yaml"]
+ok = True
+for root, _, fyles in os.walk("."):
+    if "cbmc-batch.yaml" in fyles:
+        proc = subprocess.run(cmd, cwd=root)
+        if proc.returncode:
+            ok = False
+            logging.error(
```

Comment:
Is there value to logging the return code and stderr?

Reply:
Done